spec/models/job_proxy_dispatcher_vm_storage2proxies_spec.rb

```diff
@@ -36,7 +36,7 @@
           end
 
           it "Vm#storage2proxies will exclude proxy-less hosts" do
-            @vm.storage2proxies.should be_empty
+            expect(@vm.storage2proxies).to be_empty
```

Comment:
These are @Fryguy's favorite method names  `storage2proxies`, `storage2hosts`, `proxies4job`  :laughing: 

@chrisarcand Confession, I wrote many of these job_proxy_dispatch* specs and haven't come back to the code or tests to make them cleaner.  :crying_cat_face: every time I look at them.